### DISCUSSION TITLE
how to use yasnippet-capf with company?

### DISCUSSION THREAD

**Discussion by ville-h** [elken/yasnippet-capf#14] [Q&A] [answered]
How do I use yasnippet-capf with company? I tried setting up:
```
(use-package yasnippet-capf
    :defer t

    :config
    (add-to-list 'completion-at-point-functions #'yasnippet-capf)
)
```
and I've set company-backends, with the old "working"  value commented out:
```
(setq-local company-backends
    '(
        ;; (company-capf company-yasnippet)
        (company-capf)
    )
)
```
But now I no don't get any of my snippets suggested.

**Comment by elken** [comment, marked as answer]
As mentioned in your original issue, I don't use company. I doubt it offers anything advantageous over `company-yasnippet` so if it works I'd say stick with that :smile:
